src/OSPSuite.Presentation/Presenters/Importer/IImporterReloadPresenter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Presentation.Views.Importer;
+
+namespace OSPSuite.Presentation.Presenters.Importer
+{
+   public interface IImporterReloadPresenter : IDisposablePresenter, IPresenter<IImporterReloadView>
+   {
+      void Show();
+      bool Canceled();
+      void AddDeletedDataSets(IEnumerable<string> allNames);
+      void AddNewDataSets(IEnumerable<string> except);
+      void AddOverwrittenDataSets(IEnumerable<string> intersect);
+   }
+
+   public class ImporterReloadPresenter : AbstractDisposablePresenter<IImporterReloadView, IImporterReloadPresenter>, IImporterReloadPresenter
+   {
+
+
+      public ImporterReloadPresenter(
+         IImporterReloadView view) : base(view)
+      {
+      }
+
+      public void Dispose()
+      {
```

Comment:
Do you need to defined a dispose method empty?

Reply:
removed